roles/openstack/tasks/main.yaml

```diff
@@ -0,0 +1,82 @@
+---
+# (c) Copyright IBM Corporation
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Ensure workspace exists
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    mode: "0755"
+    owner: "{{ ansible_user_id }}"
+    group: "{{ ansible_user_gid }}"
+  loop:
+    - "{{ neerali_openstack_basedir }}"
+    - "{{ neerali_openstack_artifactdir }}"
+    - "{{ neerali_openstack_logdir }}"
+  loop_control:
+    label: "{{ item }}"
+
+- name: Create networks
+  ansible.builtin.include_tasks: network.yaml
+  vars:
+    action: delete
```

Comment:
`action: delete` - and the task is `create networks`, not sure if this was the intented operation.

Reply:
sorry, i added it while testing locally but forgot to revert it.